101-cosmosdb-api-for-mongodb-account-create/azuredeploy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Can you format that JSON - i.e. SHIFT+ALT+F in VS Code, etc...  Do so for all the JSON files in the PR...

Reply:
It looks like the files are using tabs vs. spaces - std indent for the json files in the repo is 2 spaces... can you replaces the tabs?  otherwise subsequent PRs may have diffs on every line due to tab vs. space.

Reply:
Good catch, sorry about that, it's fixed now, thanks!